mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -418,18 +418,24 @@ def _get_records(
             # Frames client expects the aggregators to be a comma-separated string
             agg_funcs = ",".join(agg_funcs)
         table_path = self.tables[table]
-        df = self._frames_client.read(
-            backend=_TSDB_BE,
-            table=table_path,
-            start=start,
-            end=end,
-            columns=columns,
-            filter=filter_query,
-            aggregation_window=interval,
-            aggregators=agg_funcs,
-            step=sliding_window_step,
-            **kwargs,
-        )
+        try:
+            df = self._frames_client.read(
+                backend=_TSDB_BE,
+                table=table_path,
+                start=start,
+                end=end,
+                columns=columns,
+                filter=filter_query,
+                aggregation_window=interval,
+                aggregators=agg_funcs,
+                step=sliding_window_step,
+                **kwargs,
+            )
+        except v3io_frames.ReadError as err:
+            if "No TSDB schema file found" in str(err):
+                return pd.DataFrame()
```

Comment:
What is the `/metrics` output on a new model endpoint?
If I follow correctly, it will always show the "invocations" metric, which is incorrect.

Reply:
It will return an empty dataframe, and hence the expected `ModelEndpointMonitoringMetricNoData`.

Reply:
I refer to the `read_prediction_metric_for_endpoint_if_exists` method, which is used in the `/metrics` endpoint.
What is the `/metrics` (not `/metrics-values`) output in the mentioned case?

Reply:
> What is the `/metrics` (not `/metrics-values`) output in the mentioned case?

👆

Reply:
This question remains unanswered - it was the primary motivation for this fix.
It should be ok now. Addressing it could reduce the review time for both sides.